frontend/src/routes/manage/Realm/Content/index.tsx

```diff
@@ -115,9 +116,21 @@ const ManageContent: React.FC<Props> = ({ data }) => {
     };
 
 
-    const hasUnsavedChanges = blocks.some(block => block.editMode);
+    const editedBlock = blocks.find(block => block.editMode);
+    const hasUnsavedChanges = editedBlock !== undefined;
     useNavBlocker(hasUnsavedChanges);
 
+
+    // When a block goes into edit mode, we want to scroll it into view
+    const blockRefs = useRef(new Map<string, HTMLDivElement>());
+    useEffect(() => {
+        if (hasUnsavedChanges) {
+            const ref = blockRefs.current.get(editedBlock.id) ?? bug("unbound ref");
+            ref.scrollIntoView({ behavior: "smooth" });
```

Comment:
```suggestion
            ref.scrollIntoView({ behavior: "smooth", block: "center" });
```

I would scroll it into the middle of the view. I find it kind of weird to have the div, especially if it's not very high, to be at the very top. It gets more confusing the larger your screen is.

Reply:
Oh cool, I didn't know about that. :D